psalm.xml

```diff
@@ -0,0 +1,185 @@
+<?xml version="1.0"?>
+<psalm
+    totallyTyped="false"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="/Users/matthewbrown/Desktop/vimeo/git/cakephp/vendor/vimeo/psalm/src/config.xsd"
```

Comment:
This location won't exist on most people's computers.

Reply:
works on mine. Production-ready code.